Libraries/Text/RCTTextViewManager.m

```diff
@@ -66,19 +67,19 @@ - (UIView *)view
 
 RCT_CUSTOM_VIEW_PROPERTY(fontSize, NSNumber, RCTTextView)
 {
-  view.font = [RCTFont updateFont:view.font withSize:json ?: @(defaultView.font.pointSize)];
+  view.fontAttributes.fontSize = json ?: @(defaultView.font.pointSize);
```

Comment:
... and get rid of all custom imperative logic inside ViewManager.
We don't really need `defaultView` here.

Reply:
@shergin Are you saying we should default to passing `nil` to `fontAttributes` properties instead of forwarding whatever was in `defaultView`?

I think that will change the behavior but at least now things will be more consistent. For example, today I think if you don't specify `fontSize` during the initial render, the rendered `fontSize` will be `RCTFont's` default. If you later set the `fontSize` to `20` and then back to `nil`, I think it'll fallback to a default of `defaultView.font.pointSize` which is not necessarily the same as `RCTFont's` default so the font could be a different size now than it was when it was initially rendered.